### PR TITLE
Create transfer_file.yml

### PR DESCRIPTION
**Pull request by qqmok**

### CODE REVIEW

File: .github/workflows/transfer_file.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Transfer file
+
+# Controls when the workflow will run
+on:
+  workflow_dispatch:
+    inputs:
+      FILENAME:
+        type: string
+        default: path/filename.json
```
Comment (dweinholz) [inline .github/workflows/transfer_file.yml]: Can you change this to required and remove the default? 

A default does not make sense for this use case

**Review by dweinholz — CHANGES_REQUESTED** [review]

File: .github/workflows/transfer_file.yml (hunk 1 above)
Comment (qqmok) [reply .github/workflows/transfer_file.yml]: Is done, thanks!

File: .github/workflows/transfer_file.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+            
+      - name: Transfer file with fts3-connector
+        run: |
+          source /home/ubuntu/.API_KEY
+          echo "Calling with ${{ github.event.inputs.FILENAME }}"
+          curl -H "X-AUTH-HEADER: $API_KEY" -H "Content-Type: application/json" -X POST -d @${{ github.event.inputs.FILENAME }} "http://192.168.2.197:8080/transfer"
```
Comment (dweinholz) [inline .github/workflows/transfer_file.yml]: Please use the environment vars

File: .github/workflows/transfer_file.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  changed-files:
+    # The type of runner that the job will run on
+    runs-on: [self-hosted, fts3]
+    environment: APPROVAL_REQUIRED
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Add mask
+        run: |
+          source /home/ubuntu/.API_KEY
```
Comment (dweinholz) [inline .github/workflows/transfer_file.yml]: Please use the environment secret instead

**Review by dweinholz — CHANGES_REQUESTED** [review]

**Review by dweinholz — APPROVED** [review]

**Pull request merged**